website/content/en/preview/getting-started/getting-started-with-terraform/_index.md

```diff
@@ -160,9 +160,10 @@ module "eks" {
       # Not required nor used - avoid tagging two security groups with same tag as well
       create_security_group = false
 
-      min_size     = 1
-      max_size     = 1
-      desired_size = 1
+      # Ensure enough capacity to run 2 Karpenter pods
+      min_size     = 2
+      max_size     = 2
```

Comment:
you want max to be more than two, so it can do upgrades and replace nodes

Reply:
Update it